library/alloc/src/string.rs

```diff
@@ -1096,8 +1100,11 @@ impl String {
     #[must_use]
     #[stable(feature = "string_as_str", since = "1.7.0")]
     #[cfg_attr(not(test), rustc_diagnostic_item = "string_as_mut_str")]
-    pub fn as_mut_str(&mut self) -> &mut str {
-        self
+    #[rustc_const_unstable(feature = "const_vec_string_slice", issue = "129041")]
```

Comment:
Something odd happened here, the function was made `const` but never got listed on the tracking issue...

Reply:
Updated the tracking issue and renominated for libs-api.